build.zig.zon

```diff
@@ -3,6 +3,9 @@
 	.version = "0.0.0",
 	.paths = .{""},
 	.dependencies = .{
+		.glfw = .{
+			.path = "../glfw/cubyz-sources"
+		},
```

Comment:
You should get glfw from their official github directly.
Checking out glfw locally should not be a requirement to build Cubyz.
`.url = "https://github.com/glfw/glfw/archive/refs/tags/3.4.tar.gz"` for the latest release.

Reply:
Sure. Especially until at least Mac can cross-compile.